packages/api/src/lcd.ts

```diff
@@ -16,6 +44,7 @@ class API {
   account: Account
 
   constructor(endpoint?: string) {
+    super(endpoint)
```

Comment:
i fix the merge from master by adding this. @antho1404 can you check if it's useful